source/helpers/DataStructure.js

```diff
@@ -0,0 +1,159 @@
+const createAnswerObject = data => ({
```

Comment:
Missing JSDOC. Just a thought if this creates a case answer object? Then maybe rename this to createCaseAnswerObject :)